docs/development.md

```diff
@@ -1,10 +1,11 @@
 # Running the SHIELD Dashboard locally
 
-1. `echo DEBUG=True >> .env`
+1. `echo DEBUG=true >> .env`
```

Comment:
Ah, I can revert it, in a previous version of the PR I used the DEBUG env var for deciding which endpoint to use.